src/components/plots/styling/PointDesign.jsx

```diff
@@ -31,19 +31,22 @@ const PointDesign = (props) => {
             marks={{ 1: 1, 100: 100 }}
           />
         </Form.Item>
-        <Form.Item
-          label='Point Fill Opacity'
-        >
-          <Slider
-            value={newConfig.marker.opacity}
-            min={1}
-            max={10}
-            onChange={(value) => {
-              handleChange({ marker: { opacity: value } });
-            }}
-            marks={{ 1: 1, 10: 10 }}
-          />
-        </Form.Item>
+
+        {config.marker.showOpacity ? (
```

Comment:
Why don't you just add a check in `PlotStyling` for if config is defined? This way we don't need to check for the config in any of the plot controls which all rely on config.

Reply:
And if config is undefined, show loader on the controls, what do you think?

Reply:
This check selectively enable / disable the slider for point opacity.  if `showOpacity` is true, it shows the slider. This is the case for categorical embedding and continuous embedding

Feedback from Vicky, we want this to be disabled for volcano plot, so this control is hidden by setting `showOpacity` to `false` for volcano plot.